### DISCUSSION TITLE
Problem to show gps position

### DISCUSSION THREAD

**Discussion by dl1hrc** [OpenOrienteering/mapper#1865] [Questions & Answers]
Hi all,
I followed the instructions from here https://www.openorienteering.org/mapper-manual/pages/touch-mode.html and have connected a gps mouse to the laptop and configured it. But I couln't find the buttons for "Compass display toggle" and "Location display toggle".
My open orienteering mapper version is 0.9.4
Would be nice to get a small hint, thanks
Kind regards
Adi

**Comment by dg0yt** [comment]
Dear Adi,
at the moment, GNSS tracking is only available in touch mode. There is a big button on the left side of the home screen for toggling touch mode.
73, Kai

**Reply by dl1hrc** [reply]
Hi Kai,
thank you for your replay. I have enable "Wekzeuge->Touch-Cursor" but couldn't find a big button ont the left. Maybe I also have a problem with understanding. My goal would be to compare the map with reality on site and I would expect something like a crosshair or something similar.

 ![oom-gps](https://user-images.githubusercontent.com/8234177/106364267-0e084300-632e-11eb-97c3-e2bedfed6a0d.png)

73s de Adi

**Reply by dg0yt** [reply]
No, touch cursor is a different thing. You are still using the desktop user interface. Click menu Datei > Schließen. Then click "Touch-Modus" on the left side of the screen.

**Reply by dl1hrc** [reply]
Ok, thank you
73s de Adi
